### DISCUSSION TITLE
When changing wallet address in metamask it does not update the connected wallet in application

### DISCUSSION THREAD

**Discussion by WingsDevelopment** [rainbow-me/rainbowkit#2050] [Q&A]
Not sure if this is supported or not?

**Comment by DanielSinclair** [comment]
@WingsDevelopment Yes, this is supported via Wagmi. Ensure that you're changing the wallet that is connected to the dApp, rather than the wallet you're viewing in MetaMask. If you see any further issue, this sounds like it would be a MetaMask issue.

**Reply by WingsDevelopment** [reply]
@DanielSinclair yeah, you are correct, i wasn't using my metamask correctly! thanks
